### ISSUE TITLE
Parallel DMRG-SCF via pymolcas

### ISSUE THREAD

**Issue by dmrg-learner** [qcscine/qcmaquis#21]
Hi,

  I tried to follow https://gitlab.com/Molcas/OpenMolcas/-/issues/236
by
`pymolcas -np 4 011.input`
(https://github.com/Molcas/OpenMolcas/blob/dbb6ed0fcd4552d29700dd95d6896ed20c9e5857/test/qcmaquis/011.input)

I saw
```
()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()

                                              &DMRGSCF

                                   only a single process is used
                       available to each process: 2.0 GB of memory, 1 thread
                                             pid: 13236
()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()

```
seems only one core has been used. Any simple way to parallel DMRG-SCF? Thank you very much

**Comment by weymutht** [comment]
You should be able to specify the number of cores QCMaquis uses by setting the environment variable `OMP_NUM_THREADS`. For example, to do a calculation on 8 cores, you would set `export OMP_NUM_THREADS=8`.

**Comment by dmrg-learner** [comment]
> You should be able to specify the number of cores QCMaquis uses by setting the environment variable `OMP_NUM_THREADS`. For example, to do a calculation on 8 cores, you would set `export OMP_NUM_THREADS=8`.

Thanks. Now I see 
```

()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()

                                              &DMRGSCF

                                   only a single process is used
                       available to each process: 2.0 GB of memory, 4 threads
                                             pid: 20077
()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()
```
